spock/plugins/helpers/entities.py

```diff
@@ -134,7 +134,7 @@ def handle_spawn_player(self, event, packet):
         self.ec.entities[packet.data['eid']] = entity
         self.ec.players[packet.data['eid']] = entity
         self.event.emit('entity_spawn', {'entity': entity})
-        self.event.emit('player_spawn', entity)
+        self.event.emit('entity_player_spawn', entity)
```

Comment:
Either that, or we could rename the `clientinfo` plugin to `client` plugin.

The event names are not visible to the end user of a bot anyways, so we should follow a clear naming scheme for the programmer. This means length does not matter, but consistency does.